### DISCUSSION TITLE
Deta pricing page?

### DISCUSSION THREAD

**Discussion by ankraj077** [deta/cloud-docs#677] [Help]
Where is Deta official pricing page?

**Comment by lemonyte** [comment]
There is none, because Deta does not charge for its services.

**Reply by ankraj077** [reply]
Thanks @LemonPi314 for response. Deta should flag this page then: https://www.saasworthy.com/product/deta/pricing

 Is there any limitation on projects or number of requests for endpoint?

**Reply by lemonyte** [reply]
There is no limit on the number of projects or requests. There are some technical limitations like the 10-second execution timeout and 400 KB max size per Base row, and there is a limit of 10 GB of Drive space per account. I've just about covered all of them.
